UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.IntegrationTests/Configuration.cs

```diff
@@ -23,6 +24,7 @@ public Configuration()
             ProductType = ConfigurationRoot.GetValue<string>("FileShareService:ProductType");
             BaseUrl = ConfigurationRoot.GetValue<string>("FileShareService:BaseUrl");
             FssClientId = ConfigurationRoot.GetValue<string>("FileShareService:FssClientId");
+            MaxAttributeValuesCount = ConfigurationRoot.GetValue<int>("FileShareService:MaxAttributeValuesCount");
```

Comment:
Please check MaxAttributeValuesCount is reflected in the attribute search value count.

This can be added in existing test "WhenCallIndexOnLoad_ThenReturnList()" as:

Assert.GreaterOrEqual(Config.MaxAttributeValuesCount, showWeeklyFiles.YearAndWeekList.Count);

Reply:
Added, Thanks